interface.md

```diff
@@ -29,7 +29,7 @@ Therefore, the interface consists of:
 - `condition(::Model, ::Trace) -> ConditionedModel`
 - `decondition(::ConditionedModel) -> GenerativeModel`
 - `sample(::Model, ::Sampler = Exact(), [Int])` (from `AbstractMCMC.sample`)
-- `logdensity(::Model, ::Trace)`
+- `DensityInterface.logdensityof(::Model, ::Trace)`
```

Comment:
Maybe to be consistent with the line above:
```suggestion
- `logdensityof(::Model, ::Trace)` (from `DensityInterface.logdensityof`)
```
And maybe add and export `densityof` as well which is defined automatically?